website/docs/r/azurerm_server_vulnerability_assessment.markdown

```diff
@@ -0,0 +1,112 @@
+---
+subcategory: "Security Center"
+layout: "azurerm"
+page_title: "Azure Resource Manager: azurerm_server_vulnerability_assessment"
+description: |-
+  Adds Azure Vulnerability Assessment (Qualys) to a VM.
+---
+
+# azurerm_server_vulnerability_assessment
+
+Adds Azure Server Vulnerability Assessment (Qualys) to a VM.
+
+## Example Usage
+
+```hcl
+resource "azurerm_resource_group" "example" {
+  name     = "example-resource-group"
+  location = "West US"
+}
+
+resource "azurerm_virtual_network" "example" {
+  name                = "net-example"
+  resource_group_name = azurerm_resource_group.example.name
+  address_space       = ["192.168.1.0/24"]
+  location            = azurerm_resource_group.example.location
+}
+
+resource "azurerm_subnet" "example" {
+  name                 = "snet-example"
+  resource_group_name  = azurerm_resource_group.example.name
+  virtual_network_name = azurerm_virtual_network.example.name
+  address_prefix       = "192.168.1.0/24"
+}
+
+resource "azurerm_network_interface" "example" {
+  name                = "vm-example"
+  location            = azurerm_resource_group.example.location
+  resource_group_name = azurerm_resource_group.example.name
+
+  ip_configuration {
+    name                          = "vm-example"
+    subnet_id                     = azurerm_subnet.example.id
+    private_ip_address_allocation = "Dynamic"
+  }
+}
+
+resource "azurerm_virtual_machine" "example" {
```

Comment:
The documentation for `azurerm_virtual_machine` states

> The azurerm_virtual_machine resource has been superseded by the azurerm_linux_virtual_machine and azurerm_windows_virtual_machine resources.

Should examples in documentation be updated to one of these?

Reply:
Thanks, I have updated the documentation